docs/src/IconButton.doc.js

```diff
@@ -12,202 +12,357 @@ const card = c => cards.push(c);
       },
       {
         name: 'accessibilityHaspopup',
         type: 'boolean',
-        description:
-          'Indicates that the element has a popup context menu or sub-level menu. Populates aria-haspopup.',
-        href: 'accessibility-popup',
-      },
-      {
-        name: 'accessibilityLabel',
-        type: 'string',
-        description:
-          'String that clients such as VoiceOver will read to describe the element. Populates aria-label.',
-        href: 'accessibility-popup',
+        required: false,
+        defaultValue: null,
+        description: [
+          'Indicate that a button component controls the appearance of interactive popup elements, such as menu or dialog.',
+          'Optional with button-role buttons.',
+          'Accessibility: It populates aria-haspopup.',
+        ],
+        href: 'accessibility',
       },
       {
         name: 'bgColor',
         type:
           '"transparent" | "transparentDarkGray" | "darkGray" | "gray" | "lightGray" | "white" | "red"',
+        required: false,
         defaultValue: 'transparent',
-        required: true,
+        description: 'Primary colors to apply to the button background.',
         href: 'backgroundColorCombinations',
       },
       {
         name: 'dangerouslySetSvgPath',
         type: `{ __path: string }`,
-        description: `When using this prop, make sure that the viewbox around the SVG path is 24x24`,
+        required: false,
+        defaultValue: null,
+        description: [
+          'Define a new icon different from the built-in Gestalt icons.',
+          `When using 'dangerouslySetSvgPath', the viewbox around the SVG path must be 24x24 px`,
+        ],
+        href: '',
       },
       {
         name: 'disabled',
         type: 'boolean',
-        defaultValue: false,
-        href: 'disabledCombinations',
+        required: false,
+        defaultValue: null,
+        description:
+          'Set disabled state so IconButton looks inactive, cannot be interacted with, and actions are not available.',
+        href: 'roles',
       },
       {
         name: 'iconColor',
         type: `"darkGray" | "gray" | "red" | "white"`,
+        required: false,
         defaultValue: 'gray',
-        href: 'iconColorCombinations',
+        description: 'Primary colors to apply to the Gestalt Icon.',
+        href: 'iconColor',
       },
       {
         name: 'icon',
         type: '$Keys<typeof icons>',
-        description: `This allows type checking for a valid icon name based on the keys from the list of icons in
-        Icon.`,
+        required: false,
+        defaultValue: null,
+        description: [
+          'Add a Gestalt icon to be displayed and convey the behaviour of the button.',
+          `Accessibility: Icons are visual informacion and are not accessible for screen readers. 'accessibilityLabel' must be passed to provide sufficient context to the IconButton.`,
+        ],
+        href: '',
```

Comment:
@AlbertCarreras can you remove the empty `href`?

Reply:
it's `icon`

Reply:
DONE